### ISSUE TITLE
addition to code folding

### ISSUE THREAD

**Issue by derStephan** [cjdinger/sas-npp#1]
Hi,

first of all, thank you for the great language-file. Using it a lot. 

May I ask to add %MACRO/%MEND to code folding? 

Thanks in advance. 
Stephan

**Comment by cjdinger** [comment]
Sure -- do you have a specific code change? You can make the change and issue a pull request.  I'll review it and merge.

**Comment by Bruno-Dionne** [comment]
Hi,

Please see this SAS-NPP fork on github where this feature "macro folding"
is already implemented, including an enhanced SAS comments color syntax
(support all kinds of SAS comments ).


https://github.com/Bruno-Dionne/Notepad-UDL2---User-defined-Language-for-SAS

You will alos see all SAS Enterprise Guide color Syntax sheme duplicate in
Notepad++


There is also an older version merger waiting here (not recommmended):
https://github.com/Bruno-Dionne?tab=repositories

Have a nice day.

Bruno Dionne
Université Laval
Québec, Qc, Canada

**Comment by Bruno-Dionne** [comment]
Sorry, forgot to mention that it's based on cjdinger/sas-npp work and many
others.

**Comment by derStephan** [comment]
Thanks, I wil take a look. I suppose, I should start with "default", right?

**Comment by Bruno-Dionne** [comment]
Yes default is a wise choice if you use the Enterprise Guide Editor default
like me.

Feel free to suggest any change, correction or new features.

Bruno Dionne

Le ven. 18 janv. 2019 02 h 33, derStephan <notifications@github.com> a
écrit :
